### PR TITLE
Create V1 of front-end for webring

### PR DESCRIPTION
**Pull request by saymaakhan**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	if ($(window).width() > 768){		
+    $(".collapse").addClass("in");    
+	}	
+});
+
+const charactersList = document.getElementById('charactersList');
+const searchBar = document.getElementById('searchBar');
+let ece_students = [
+    {
+    "name": "Sayma Khan",
```
Comment (adshayanB) [inline js/app.js:74]: Formatting

**Review by adshayanB — CHANGES_REQUESTED** [review]
Two small nits. 
1. Just a small formatting issue.
2. Could we deploy this on GitHub pages to see if it deploys correctly.

File: js/app.js (hunk 1 above)
Comment (saymaakhan) [reply js/app.js]: formatted with prettier

**Review by adshayanB — APPROVED** [review]
LGTM 🎸

**Pull request merged**